schema/ListedLicense.xsd

```diff
@@ -59,7 +59,7 @@
 	</complexType>
 	<complexType name="notesType" mixed="true">
 		<choice minOccurs="1" maxOccurs="unbounded">
-			<element name="p" type="tns:formattedTextType"/>
+			<element name="p" type="tns:formattedAltTextType"/>
```

Comment:
Notes does not support alternative or optional text - should this be formattedFixTextType?  If we change it, will it cause a problem with the other elements named "p" in other elements?

Reply:
> If we change it, will it cause a problem with the other elements named "p" in other elements?

I din't think so.  Will rebase tonight.